components/admin_console/system_users/list/__snapshots__/system_users_list.test.jsx.snap

```diff
@@ -12,6 +12,7 @@ exports[`components/admin_console/system_users/list should match default snapsho
         "doPasswordReset": [Function],
         "enableUserAccessTokens": false,
         "experimentalEnableAuthenticationTransfer": false,
+        "isDisabled": undefined,
         "mfaEnabled": false,
       }
     }
```

Comment:
I think the related test file for this should be updated and a value for `isDisabled` should be passed

Reply:
Done.